src/commons/application/types/SessionTypes.ts

```diff
@@ -118,7 +117,6 @@ export type SessionState = {
   readonly gradings: Map<number, GradingQuery>;
   readonly notifications: Notification[];
   readonly googleUser?: string;
-  readonly githubAssessment?: MissionRepoData;
   readonly githubOctokitObject: { octokit: Octokit | undefined };
```

Comment:
I'm leaving this object here just in case the folders team would require it (to minimise conflicts); but if it turns out to not be needed then it can be removed in the future.